_posts/2017-08-15-magick-10.md

```diff
@@ -0,0 +1,164 @@
+---
+name: magick_release_10
+layout: post_discourse
+title: "Magick 1.0: 🎩 ✨🐇 Advanced Graphics and Image Processing in R "
+date: 2017-08-15
+authors:
+  - name: Jeroen Ooms
+categories: blog
+tags:
+- r
+- packages
+- software
+- images
+- magick
+---
+
+
+Last week version 1.0 of the [magick](https://cran.r-project.org/web/packages/magick/index.html) package has appeared on CRAN: an ambitious effort to modernize and simplify high quality image processing in R. This R package builds upon the [Magick++ STL](https://www.imagemagick.org/Magick++/STL.html) which exposes a powerful C++ API to the famous ImageMagick library.
```

Comment:
insert comma, delete "has" like this:

"Last week, version 1.0 of the [magick](https://cran.r-project.org/web/packages/magick/index.html) package appeared on ..."